pep-0517.txt

```diff
@@ -285,11 +231,66 @@ specifies UTF-8 based file names. This is not yet the default for the tarfile
 module shipped with Python 3.6, so backends using the tarfile module need to
 explicitly pass ``format=tarfile.PAX_FORMAT``.
 
-This must return the basename (not the full path) of the ``.tar.gz`` file it
+The hook must return the basename (not the full path) of the ``.tar.gz`` file it
 creates, as a unicode string.
 
-Mandatory, but it may not succeed in all situations: for instance, some tools
-can only build an sdist from a VCS checkout.
+Frontends performing a local installation may want to produce an sdist as an
+intermediate, but are advised to be prepared to use a fallback if it fails, as
+e.g. some backends may require version control tools to build an sdist.
+
+Optional hooks
+==============
+
+  def get_requires_for_build_wheel(config_settings={}):
+      ...
+
+This hook MUST return an additional list of strings containing PEP 508
+dependency specifications, above and beyond those specified in the
+``pyproject.toml`` file, to be installed when calling the ``build_wheel`` or
+``prepare_metadata_for_build_wheel`` hooks.
+
+Example::
+
+  def get_requires_for_build_wheel(config_settings):
+      return ["wheel >= 0.25", "setuptools"]
+
+If not defined, the default implementation is equivalent to ``return []``.
+
+::
+
+  def prepare_metadata_for_build_wheel(metadata_directory, config_settings={}):
+      ...
+
+Must create a ``.dist-info`` directory containing wheel metadata
+inside the specified ``metadata_directory`` (i.e., creates a directory
+like ``{metadata_directory}/{package}-{version}.dist-info/``. This
+directory MUST be a valid ``.dist-info`` directory as defined in the
+wheel specification, except that it need not contain ``RECORD`` or
+signatures. The hook MAY also create other files inside this
+directory, and a build frontend MUST ignore such files; the intention
```

Comment:
I think this needs to say "MUST preserve, but otherwise ignore, such files". Otherwise it could be interpreted as meaning "if you copy the metadata to a different directory, omit any files you don't recognise", which is the exact opposite of what we want.